src/components/keyboard/keyboard.js

```diff
@@ -285,25 +284,6 @@ class Keyboard extends Component {
       return {preventDefault: true, payload: null};
     }
   };
-
-  /**
-   * toggles the shell hover state
-   *
-   * @returns {void}
-   * @memberof Keyboard
-   */
-  toggleHoverState(): void {
-    if (this._hoverTimeout !== null) {
-      clearTimeout(this._hoverTimeout);
-      this._hoverTimeout = null;
-    }
-    this.props.updatePlayerHoverState(true);
-    this.props.notifyHoverChange({hover: true});
```

Comment:
What about the hover event? does it still be dispatched?

Reply:
Correct, fixed.